File: src/b2b/vendor-onboarding-service/src/main/java/com/enterprise/vendoronboardingservice/HealthController.java
package com.enterprise.vendoronboardingservice;

import org.springframework.web.bind.annotation.*;
import java.util.Map;

@RestController
public class HealthController {
    @GetMapping("/healthz")
    public Map<String, String> health() {
        return Map.of("status", "ok");
    }

    @GetMapping("/metrics")
    public String metrics() {
        return "# placeholder metrics\n";
    }
}

File: src/b2b/purchase-requisition-service/src/main/kotlin/com/enterprise/purchaserequisitionservice/HealthController.kt
package com.enterprise.purchaserequisitionservice

import org.springframework.web.bind.annotation.*

@RestController
class HealthController {
    @GetMapping("/healthz")
    fun health() = mapOf("status" to "ok")

    @GetMapping("/metrics")
    fun metrics() = "# placeholder metrics\n"
}
